rpcs3/util/fixed_typemap.hpp

```diff
@@ -299,21 +299,22 @@ namespace stx
 			}
 
 			// Get actual number of created objects
-			u32 _max = 0;
+			std::vector<usz> types_to_destroy;
```

Comment:
Althogh this is probably not needed, there must be another way to get the typeinfo index.

Reply:
it's wrong anyway, since it's the wrong order.

Reply:
probably fixed now